app-crypt/acme-sh/acme-sh-2.7.9.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN/-/.}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="An ACME Shell script"
+HOMEPAGE="https://github.com/Neilpang/acme.sh"
+SRC_URI="https://github.com/Neilpang/${MY_PN}/archive/${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="apache2 +cron nginx libressl standalone"
+
+RDEPEND="net-misc/curl
+	|| ( net-analyzer/netcat6 net-analyzer/openbsd-netcat )
+	sys-apps/iproute2
+	sys-apps/net-tools
+	sys-apps/util-linux
+	apache2? ( www-servers/apache:2 )
```

Comment:
I do not see how all these flags are affecting installation, the dependencies should not be flagged, use `|| ( server1 server2)` as well

Reply:
This is not about installation, this is about script runtime.
I for sure do not want to have apache, nginx, socat as requirements to install this script when during runtime I will use just one of them.
That's why there is apache?, nginx?, standalone? to have a possibility to install only software that is going to be used.
Or you would like to see || ( www-servers/nginx:0 www-servers/apache:2 net-misc/socat ) ?

Reply:
well, then  `any-of-many` dependencies logic would be better, yes

Reply:
|| ( www-servers/nginx:0 www-servers/apache:2 net-misc/socat ) added.